src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Machine {

```

Comment:
The class `Machine` should be declared as public to ensure that it can be accessed from other packages if required.